symfit/core/models.py

```diff
@@ -933,7 +933,7 @@ def __init__(self, model_dict, initial, *lsoda_args, **lsoda_kwargs):
         self.lsoda_args = lsoda_args
         self.lsoda_kwargs = lsoda_kwargs
 
-        sort_func = lambda symbol: str(symbol)
+        sort_func = str
```

Comment:
good change, but I've sinse learned that it would be even more in keeping with sympy to use `lambda symbol: symbol.name`, because calling string is considered an anti-pattern whereas all symbols have a name. I've therefore started doing that in new code, so it might be worth updating this as well.

Reply:
Seems less readable to me, is there a strong argument for preferring operator.attrgetter?

Reply:
It should be faster (I think). Readability is in part a matter of getting used to it. I personally prefer the attrgetter over dissecting a lambda.

Reply:
I'd prefer the lambda, but it's not something I would stop the merging for so choose your favorite. I guess after this comment has been addressed we can merge this pr.

Reply:
My preference is the attrgetter, so squash and merge at your convenience.